### ISSUE TITLE
Tracks in Top 50 not queuing the next track in the list but plays the next queued track in Stream

### ISSUE THREAD

**Issue by iRuisu** [Soundnode/soundnode-app#708]
Only the track you choose to play in the Top 50 will play, once that track ends it jumps back to the Stream queue and plays which ever track was up next within it. So pretty much the Top 50 doesn't seem to be queuing any of its track list.

**Comment by wb9688** [comment]
@Lewis it does, but only if your queu is empty...

**Comment by titenis** [comment]
can confirm this bug still exists

**Comment by weblancaster** [comment]
Yes by design that's how should work if you want the top 50 to be entirely on your queue list you will have to start fresh.. e.g once you start the app navigate to top 50 and play a track.

That being said the design isn't great :( so I started working on a fix and will be available in the next release.

thanks for the feedback

**Comment by wb9688** [comment]
@weblancaster can you add a remove everything from queue button?

**Comment by weblancaster** [comment]
Yes @wb9688
